{
  "modified": "0001-01-01T00:00:00Z",
  "published": "2016-03-28T00:00:00Z",
  "schema_version": "1.5.0",
  "id": "PSF-2016-9",
  "summary": "Issue #26657: HTTP server directory traversal",
  "details": "Fix directory traversal vulnerability with ``http.server`` and\n``SimpleHTTPServer`` on Windows.\n\nRegression of Python 3.3.5.\n\nPython issue reported at 2016-03-14.\n",
  "affected": [
    {
      "ranges": [
        {
          "type": "GIT",
          "events": [
            {
              "introduced": "0"
            },
            {
              "fixed": "0cf2cf2b7d726d12a6046441e4067d32c7dd4feb"
            },
            {
              "fixed": "6f6bc1da8aaae52664e7747e328d26eb59c0e74f"
            },
            {
              "fixed": "7b92f9fa47df754b50c64aac84cf1c09693571af"
            },
            {
              "fixed": "d274b3f1f1e2d8811733fb952c9f18d7da3a376a"
            }
          ],
          "repo": "https://github.com/python/cpython"
        }
      ]
    }
  ],
  "references": [
    {
      "type": "REPORT",
      "url": "https://bugs.python.org/issue26657"
    }
  ],
  "database_specific": {
    "cwe_ids": []
  }
}
